examples/ClientExample.py

```diff
@@ -104,8 +104,10 @@ def main_test():
         do_simple_execute(client, 'INSERT VERTEX person(name, age) VALUES 1:(\'Bob\', 10)')
         do_simple_execute(client, 'INSERT VERTEX person(name, age) VALUES 2:(\'Lily\', 9)')
         do_simple_execute(client, 'INSERT VERTEX person(name, age) VALUES 3:(\'Tom\', 10)')
+        do_simple_execute(client, 'INSERT VERTEX person(name, age) VALUES 4:(\'Jerry\', 13);INSERT VERTEX person(name, age) VALUES 5:(\'John\', 11)')
```

Comment:
add  example `INSERT VERTEX person(name, age) VALUES 6:(....), 7:(....)` and introduce the different

Reply:
> add example `INSERT VERTEX person(name, age) VALUES 6:(....), 7:(....)` and introduce the different

Should the differences between the two methods be explained in code comments or README?

Reply:
Put it in code comments